src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -144,6 +144,10 @@ public bool CanRetry
             }
         }
 
+        public HttpConnectionPool Pool => _pool;
```

Comment:
I don't think this is used anymore now, is it?

Reply:
Nope